backend/src/api/auth/auth.controller.js

```diff
@@ -0,0 +1,74 @@
+const bcrypt = require('bcrypt');
+const UserModel = require('../users/user.model');
+const AppError = require('../errors/appError');
+const jwt = require('jsonwebtoken');
+const path = require('path');
+
+require('dotenv').config({ path: path.join(__dirname, '../../../src/.env') });
+
+exports.createNewUser = async (req, res, next) => {
+  const { username, email, password } = req.body;
+  const passwordHash = await bcrypt.hash(
+    password,
+    parseInt(process.env.SALT_ROUNDS),
+  );
+  const existUser = await UserModel.findOne({ email });
```

Comment:
`existingUser`. Please fix this everywhere else

Reply:
I corrected